junyub/code/inference.py

```
@@ -0,0 +1,84 @@
+                        if f.lower().endswith(('.jpg', '.png', '.jpeg'))])
+    test_ids = [f for f in test_files]
+    test_df = pd.DataFrame({'ID': test_ids})
+
+    test_ds = MultiRecipeDataset(
+        test_df, TEST_IMG_DIR, label_col=None, img_size=IMG_SIZE, recipes=None, is_test=True
+    )
+    test_loader = DataLoader(
+        test_ds, batch_size=BATCH_SIZE, shuffle=False, num_workers=NUM_WORKERS, pin_memory=True
+    )
+    weight_paths = sorted(glob.glob(WEIGHTS_GLOB))
+    assert len(weight_paths) > 0, '가중치(.pt) 파일을 찾을 수 없습니다.'
```

Ruff (0.12.2)

Use of `assert` detected

(S101)